bot/persistence/database_connector.py

```diff
@@ -1,37 +1,34 @@
-"""Contains logic for connecting to and manipulating the database.
-
-Todo:
-    * Add queries for db tables once the db design is clear.
-"""
+"""Contains logic for connecting to and manipulating the database."""
 
 from sqlite3 import Error
-from typing import List, Union
-from .queries import INSERT_PROPERTY_QUERY, GET_PROPERTY_QUERY
+from typing import List
+from bot.moderation import ModmailStatus
+from bot.persistence import queries
 from .database_manager import DatabaseManager
 
 
-class DatabaseConnector():
+class DatabaseConnector:
     """Class used to communicate with the database.
 
     The database is created and initialized using the __init__ method. The other methods support getting or adding
     properties to the database.
     """
-
-    _db_file: Union[str, None] = None
-
     def __init__(self, db_file: str, init_script=None):
         """Create a database connection to a SQLite database and create the default tables form the SQL script in
         init_db.sql.
 
         Args:
             db_file (str): The filename of the SQLite database file.
-            init_script (Union[str, None]): Optional SQL script filename that will be run when the method is called.
+            init_script (Optional[str]): Optional SQL script filename that will be run when the method is called.
         """
+        if db_file is None:
+            raise Error("Database filepath and/or filename hasn't been set.")
+
         self._db_file = db_file
         with DatabaseManager(self._db_file) as db_manager:
             if init_script is not None:
-                queries = self.parse_sql_file(init_script)
-                for query in queries:
+                queries_ = self.parse_sql_file(init_script)
```

Comment:
Because `queries` was overshadowing our `queries.py`. This didn't really affect our code but IntelliJ was whining about it. That's why I chose to add a trailing underscore because the name is fitting and I wanted to avoid this naming conflict.